7.7.поиск_ошибок_и_ревью_кода/7.7.4.py

```diff
@@ -0,0 +1,13 @@
+mx = - pow(10, 6)      
+s = 0
+for i in range(1, 11): 
```

Comment:
Не используешь переменную цикла - называешь ее `_`.
```suggestion
for _ in range(1, 11): 
```